src/ProjectTemplates/BlazorTemplates.Tests/template-baselines.json

```diff
@@ -0,0 +1,1282 @@
+{
```

Comment:
A lot of the things in this file don't appear to be specific to Blazor. Should this be trimmed to just the Blazor tests?

Reply:
It can be. I was trying to avoid making too many changes considering the temporary nature of this PR.